testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveSparkCompatibility.java

```diff
@@ -197,6 +197,147 @@ public void testReadSparkCreatedTable(String sparkTableFormat, String expectedTr
         };
     }
 
+    @Test(groups = {HIVE_SPARK, PROFILE_SPECIFIC_TESTS})
+    public void testReadTrinoCreatedOrcTable()
+    {
+        testReadTrinoCreatedTable("using_orc", "ORC");
+    }
+
+    @Test(groups = {HIVE_SPARK, PROFILE_SPECIFIC_TESTS})
+    public void testReadTrinoCreatedParquetTable()
+    {
+        testReadTrinoCreatedTable("using_parquet", "PARQUET");
+    }
+
+    @Test(groups = {HIVE_SPARK, PROFILE_SPECIFIC_TESTS})
+    public void testReadTrinoCreatedParquetTableWithNativeWriter()
+    {
+        onTrino().executeQuery("SET SESSION " + TRINO_CATALOG + ".experimental_parquet_optimized_writer_enabled = true");
+        // TODO (https://github.com/trinodb/trino/issues/7953) Native Parquet Writer writes Parquet V2 files that are not compatible with Spark's vectorized reader
+        onSpark().executeQuery("SET spark.sql.parquet.enableVectorizedReader = false");
```

Comment:
... however, i would rather have this line removed, and `io.trino.tempto.assertions.QueryAssert#assertQueryFailure` used instead

i think testing with `SET spark.sql.parquet.enableVectorizedReader = false` is simply not interesting use case.
and i don't want to have to remember to update the test, so that it becomes useful. let's make it useful from  the start

Reply:
Why is it not better to test that something is working as expected rather than testing that something is definitely broken? If we assert a query failure we could introduce a ton of downstream issues. This is the only working example of cross compatibility we have. I would much rather keep it and have the removal of this constraint be a fast follow.

Reply:
>  If we assert a query failure we could introduce a ton of downstream issues. 

what issues do you have in mind?

> Why is it not better to test that something is working as expected rather than testing that something is definitely broken? 

testing something is broken is just keeping already written test code, with automated todo to enable it, for the future

testing for a case that i personally subjectively unscientifically believe as uninteresting (enableVectorizedReader = false), doesn't add value in my eyes.
do spark users commonly disable the vectorized reader? (@hashhar do you know?)